crates/sui-config/src/node.rs

```diff
@@ -284,8 +284,8 @@ impl Default for AuthorityStorePruningConfig {
     fn default() -> Self {
         Self {
             objects_num_latest_versions_to_retain: u64::MAX,
-            objects_pruning_period_secs: u64::MAX,
-            objects_pruning_initial_delay_secs: u64::MAX,
+            objects_pruning_period_secs: 24 * 60 * 60,
+            objects_pruning_initial_delay_secs: 60 * 60,
             num_latest_epoch_dbs_to_retain: usize::MAX,
             epoch_db_pruning_period_secs: u64::MAX,
```

Comment:
it doesn't enable old pruning by default. I had just to change value from u64::MAX to avoid overflow error on tick.
Previously we were disabling pruning actor completely if objects_num_latest_versions_to_retain == u64::MAX so corresponding time interval was never initiated. We need to keep actor running for live pruning. 
So in this PR we initiate interval, but ignore its `tick` if old pruning is disabled.